### ISSUE TITLE
'data' directory in the preprocess_nombank.py 

### ISSUE THREAD

**Issue by egumasa** [CogComp/SRL-English#1]
Hi Thank you very much for sharing this wonderful repo. I am interested in SRL at different linguistic levels on one pipeline, so this is very nice working example for me. 

I followed the instruction and at preprocessing_nombank.py, where we need to set the directory for `root_pathp`, `nom file_pathp`, `words_pathp`, and `data` (this is around line 209). 

I figured out the first three, but I was not sure what directory I should be feeding for the `data`. I am currently having `nltk_data` directory, but I assume from the code that this has to do with wsj data. 
Could you kindly tell me what this data is and required internal structure of this directory? 

Also, I was wondering if there are any active links to the pretrained models.

Just in case, here is the excerpt of my modified paths:
```
root_pathp = FileSystemPathPointer('/Users/masakieguchi/nltk_data/corpora/nombank.1.0')
nomfile_pathp = FileSystemPathPointer('/Users/masakieguchi/nltk_data/corpora/nombank.1.0/sorted_nombank.1.0')
framefiles = '.*\.xml'
words_pathp = FileSystemPathPointer('/Users/masakieguchi/nltk_data/corpora/nombank.1.0/nombank.1.0.words')
```
```
data = '/Users/masakieguchi/nltk_data'
instances = reader.instances()

train_filepath = 'train.srl' #preprocess_nombank/train.srl
devel_filepath = 'development.srl' #preprocess_nombank/development.srl
test_filepath = 'test.srl' #preprocess_nombank/test.srl
```
I removed `preprocess_nombank` dir from the setting as I am already in that directory when I followed the readme instruction.
The `sorted_nombank1.0` seems to be working.

Thank you so much in advance.

**Comment by celine-lee** [comment]
Hi! Glad to hear that this is a helpful example repository.

You are correct in assuming that the `data` variable has to do with the WSJ data. I used this repository to feed in the 'data' variable: https://github.com/Hai-Pham/ALGNLP_2
I cloned it, then used the path to its 'data' folder for the `data` variable. If you have another form of the dataset structured the same way, that should work too, but I have only tested it with this one. 

Hope that helps!
